core-ui/src/resources/APIRules/HostAndSubdomain.js

```diff
@@ -15,7 +15,7 @@ function getSelectedHost(host, hosts) {
     const hostMatch = hosts
       .filter(hasWildcard)
       .map(h => h.replace('*', ''))
-      .find(h => host.endsWith(h));
+      .find(h => host?.endsWith(h));
```

Comment:
The `host` variable always exists here. If it's not, the empty string is returned in the line 8.